rotkehlchen/accounting/constants.py

```diff
@@ -1,2 +1,183 @@
 FREE_REPORTS_LOOKUP_LIMIT = 20
+
+
+DEFAULT_EVENT_CATEGORY_MAPPINGS = {
+    HistoryEventType.SPEND: {
+        HistoryEventSubType.FEE: EventCategory.GAS,
+        HistoryEventSubType.PAYBACK_DEBT: EventCategory.REPAY,
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.LIQUIDATE: EventCategory.LIQUIDATE,
+        HistoryEventSubType.NONE: EventCategory.SEND,
+    },
+    HistoryEventType.RECEIVE: {
+        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.RECEIVE_WRAPPED: EventCategory.RECEIVE,
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.RECEIVE,
+        HistoryEventSubType.AIRDROP: EventCategory.AIRDROP,
+        HistoryEventSubType.REWARD: EventCategory.CLAIM_REWARD,
+        HistoryEventSubType.NONE: EventCategory.RECEIVE,
+        HistoryEventSubType.NFT: EventCategory.RECEIVE,
+    },
+    HistoryEventType.INFORMATIONAL: {
+        HistoryEventSubType.APPROVE: EventCategory.APPROVAL,
+        HistoryEventSubType.GOVERNANCE: EventCategory.GOVERNANCE,
+        HistoryEventSubType.DEPLOY: EventCategory.DEPLOY,
+        HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
+        HistoryEventSubType.PLACE_ORDER: EventCategory.PLACE_ORDER,
+        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,
+    },
+    HistoryEventType.TRANSFER: {
+        HistoryEventSubType.NONE: EventCategory.TRANSFER,
+        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE,
+    },
+    HistoryEventType.TRADE: {
+        HistoryEventSubType.SPEND: EventCategory.SWAP_OUT,
+        HistoryEventSubType.RECEIVE: EventCategory.SWAP_IN,
+    },
+    HistoryEventType.WITHDRAWAL: {
+        HistoryEventSubType.NONE: EventCategory.WITHDRAW,
+        HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
+        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE,
+    },
+    HistoryEventType.DEPOSIT: {
+        HistoryEventSubType.NONE: EventCategory.DEPOSIT,
+        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
+        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE,
+        HistoryEventSubType.PLACE_ORDER: EventCategory.PLACE_ORDER,
+    },
+    HistoryEventType.MIGRATE: {
+        HistoryEventSubType.SPEND: EventCategory.SEND,
+        HistoryEventSubType.RECEIVE: EventCategory.RECEIVE,
+    },
+    HistoryEventType.STAKING: {
+        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
+        HistoryEventSubType.REWARD: EventCategory.RECEIVE,
+        HistoryEventSubType.RECEIVE_WRAPPED: EventCategory.RECEIVE,
+        HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.FEE: EventCategory.FEE,
+        HistoryEventSubType.MEV_REWARD: EventCategory.MEV,
+        HistoryEventSubType.BLOCK_PRODUCTION: EventCategory.CREATE_BLOCK,
+    },
+}
+
+EVENT_CATEGORY_DETAILS = {
```

Comment:
What is the reason for moving this mapping?

Reply:
Doesn't contain only evm default events but also defaults for eth staking and also we have some events used by exchanges